pkg/lib/authenticationflow/accept.go

```diff
@@ -138,7 +138,19 @@ func accept(ctx context.Context, deps *Dependencies, flows Flows, inputFn func(i
 		}
 
 		if errors.Is(err, ErrBotProtectionVerificationSuccess) {
-			err = nil
+			uw, isJoinedError := err.(interface{ Unwrap() []error })
+			if isJoinedError {
+				errs := uw.Unwrap()
```

Comment:
Use errors.Is or errors.As

Reply:
As per offline discussion, will use newly added `errorutil.Partition`

ref 

```go
func Partition(err error, predicate func(err error) bool) (matched error, notMatched error) {
	var _matched []error
	var _notMatched []error
	Unwrap(err, func(err error) {
		b := predicate(err)
		if b {
			_matched = append(_matched, err)
		} else {
			_notMatched = append(_notMatched, err)
		}
	})
	return errors.Join(_matched...), errors.Join(_notMatched...)
}
```

Reply:
One thing you need to pay attention. After partition, the order of the error should stay the same.